ddtrace/tracer.py

```diff
@@ -819,6 +819,24 @@ def write(self, spans):
         if spans is not None:
             self._writer.write(spans=spans)
 
+    def get_tag(self, key):
```

Comment:
Not a fan of introducing these APIs just yet. There is already `set_tags` which one can use instead.

Reply:
Should I make get tag internal or just use `Tracer._tags.get()` in our tests

Reply:
yeah I think it's ok to access the internal for the tests